### PR TITLE
add currency comma format

### PR DESCRIPTION
**Pull request by jdnazario**
Adding comma and decimal number in the currency format to have a friendly view

### CODE REVIEW

File: src/components/project/ProjectDetails.vue [hunk 1]
```diff
@@ -59,8 +59,8 @@
         <p class="label">Target date</p>
         <!-- <target-date-card :contractPrice="project.contract_price" :estimatedCost="project.est_cost" :date="date" /> -->
         <target-date-card
-          :contractPrice="this.targetDates.length > 0? targetDate.contract_price : 0"
-          :estimatedGP="this.targetDates.length > 0? targetDate.est_gp: 0"
+          :contractPrice="this.targetDates.length > 0? targetDate.contract_price.toFixed(2).replace(/(\d)(?=(\d{3})+(?!\d))/g, '$1,'): '0.00'"
+          :estimatedGP="this.targetDates.length > 0? targetDate.est_gp.toFixed(2).replace(/(\d)(?=(\d{3})+(?!\d))/g, '$1,'): '0.00'"
```
Comment (mhackyu) [inline src/components/project/ProjectDetails.vue]: @jdnazario , kindly use the currencyFormat filter from `@/utils/format` then change the code to 
`targetDate.contract_price | currencyFormat`

File: src/components/target-date/TargetDateListItem.vue [hunk 2]
```diff
@@ -40,6 +40,12 @@ export default {
   computed: {
     dateTargetDate() {
       return dateFormat(this.targetDate.target_date);
+    },
+    contractPriceTargetDate() {
+      return this.targetDate.contract_price.toFixed(2).replace(/(\d)(?=(\d{3})+(?!\d))/g, '$1,');
+    },
+    estimateGPTargetDate() {
+      return this.targetDate.est_gp.toFixed(2).replace(/(\d)(?=(\d{3})+(?!\d))/g, '$1,');
```
Comment (mhackyu) [inline src/components/target-date/TargetDateListItem.vue]: @jdnazario , kindly use the currencyFormat filter from `@/utils/format` then change the code to 
`{{ contractPriceTargetDate | currencyFormat }}` and `{{ estimateGPTargetDate | currencyFormat }}`

File: src/views/Opportunity.vue [hunk 3]
```diff
@@ -62,10 +62,10 @@ import Config from "@/config";
 export default {
   computed: {
     totalEst_gp() {
-      return this.projects.reduce((acc, item) => acc + item.est_gp, 0);
+      return this.projects.reduce((acc, item) => acc + item.est_gp, 0).toFixed(2).replace(/(\d)(?=(\d{3})+(?!\d))/g, '$1,');
```
Comment (mhackyu) [inline src/views/Opportunity.vue]: @jdnazario  , kindly use the currencyFormat filter from `@/utils/format`

File: src/views/Opportunity.vue [hunk 4]
```diff
@@ -62,10 +62,10 @@ import Config from "@/config";
...
     },
     totalContract_price() {
-      return this.projects.reduce((acc, item) => acc + item.contract_price, 0);
+      return this.projects.reduce((acc, item) => acc + item.contract_price, 0).toFixed(2).replace(/(\d)(?=(\d{3})+(?!\d))/g, '$1,');
```
Comment (mhackyu) [inline src/views/Opportunity.vue]: @jdnazario  , kindly use the currencyFormat filter from `@/utils/format`

**Comment by jdnazario** [comment]
Sir mark ano po ilalagay sa mysql para makuha ko po yung username may
nilagay po kasi si sir lorry para makapaglogin po ako

**Comment by mhackyu** [comment]
Approved! Thanks, @jdnazario 👍

**Pull request merged**